### ISSUE TITLE
Wrong react version when install package

### ISSUE THREAD

**Issue by anthonysavchenko** [echowaves/expo-cached-image#13]
When I'm trying to install `expo-cached-image` with `npx expo install expo-cached-image` command to my Expo project (latest SDK version 46) I'm getting this error

```
PS F:\Code\cinema-mobile-app> npx expo install expo-cached-image
› Installing 1 other package using npm
> npm install --save expo-cached-image
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: cinema-mobile-app@1.0.0
npm ERR! Found: react@18.0.0
npm ERR! node_modules/react
npm ERR!   react@"18.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"18.2.0" from expo-cached-image@46.0.0
npm ERR! node_modules/expo-cached-image
npm ERR!   expo-cached-image@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See C:\Users\as\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\as\AppData\Local\npm-cache\_logs\2022-08-17T07_38_33_619Z-debug-0.log

Error: npm exited with non-zero code: 1
Error: npm exited with non-zero code: 1
    at ChildProcess.completionListener (F:\Code\cinema-mobile-app\node_modules\@expo\spawn-async\build\spawnAsync.js:52:23)
    at Object.onceWrapper (node:events:640:26)
    at ChildProcess.emit (node:events:520:28)
    at ChildProcess.cp.emit (F:\Code\cinema-mobile-app\node_modules\@expo\spawn-async\node_modules\cross-spawn\lib\enoent.js:34:29)
    at maybeClose (node:internal/child_process:1092:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5)
    ...
    at spawnAsync (F:\Code\cinema-mobile-app\node_modules\@expo\spawn-async\build\spawnAsync.js:17:21)
    at NpmPackageManager._runAsync (F:\Code\cinema-mobile-app\node_modules\@expo\package-manager\build\NodePackageManagers.js:133:51)
    at NpmPackageManager.addWithParametersAsync (F:\Code\cinema-mobile-app\node_modules\@expo\package-manager\build\NodePackageManagers.js:75:24)    
    at installPackagesAsync (F:\Code\cinema-mobile-app\node_modules\@expo\cli\build\src\install\installAsync.js:82:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

It seems the `expo-cached-image` package depends on React 18.2.0 although the latest Expo SDK is providing React 18.0.0 only

**Comment by dmitryame** [comment]
Just published 46.0.1 patch release, let me know if it works.

**Comment by anthonysavchenko** [comment]
Installed with no problem now, thanks

By the way, as far as I see there is no `placeholder` in .d.ts file. And one more question, how do you use it for typescript project?
